<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThrusterDiag</class>
 <widget class="QDialog" name="ThrusterDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1468</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="customPlot_thruster" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot_thruster2" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>391</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot_thruster3" native="true">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>431</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
